package.json

```diff
@@ -16,10 +16,16 @@
   "bugs": {
     "url": "https://github.com/SU-SWS/decanter/issues"
   },
+  "dependencies": {
+    "bourbon": "^4.2.7",
+    "bourbon-neat": "^1.8.0",
```

Comment:
This is an obsolete version and completely changes in 2.0. This needs to be updated and the grid-system will need to adjust to accommodate.